### PR TITLE
fix-connect-calendar-minor-issues

### PR DESCRIPTION
**Pull request by alannnc**
## What does this PR do?

- Fix a visual bug that when redirected from apps/installed both components (List and loading skeleton) were displaying at the same time.
- Added cookie SSR redirection on new calendar list page for v2

Fixes # (issue)

- [X] Bug fix (non-breaking change which fixes an issue)

## How should this be tested?

- [ ] Do the getting-started flow and now you should be redirected to step 2.
- [ ] While TRPC is loading after being redirected to step 2, you should only be able to see either loading or calendar list.

## Checklist

- I haven't added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: apps/web/pages/v2/apps/installed/[category].tsx [hunk 1]
```diff
@@ -202,3 +203,24 @@ export default function InstalledApps() {
     </InstalledAppsLayout>
   );
 }
+
+// Server side rendering
+export async function getServerSideProps(ctx: AppGetServerSidePropsContext) {
```
Comment (alannnc) [inline apps/web/pages/v2/apps/installed/[category].tsx:208]: Same code as v1

File: apps/web/components/getting-started/steps-views/ConnectCalendars.tsx [hunk 2]
```diff
@@ -31,7 +31,8 @@ const ConnectedCalendars = (props: IConnectCalendarsProps) => {
   return (
     <>
       {/* Already connected calendars  */}
-      {firstCalendar &&
+      {!queryConnectedCalendars.isLoading &&
+        firstCalendar &&
```
Comment (alannnc) [inline apps/web/components/getting-started/steps-views/ConnectCalendars.tsx:35]: This will prevent that calendar list displays while trpc is loading

**Review by pumfleet — APPROVED** [review]
Code looks good to me, haven't tested

File: apps/web/pages/v2/apps/installed/[category].tsx [hunk 3]
```diff
@@ -209,14 +225,4 @@ export const getStaticProps: GetStaticProps = (ctx) => {
       category: params.data.category,
     },
   };
-};
-
-export const getStaticPaths: GetStaticPaths = () => {
```
Comment (hariombalhara) [inline apps/web/pages/v2/apps/installed/[category].tsx:214]: cc @leog. This has to go because of SSR requirement.

**Review by hariombalhara — APPROVED** [review]

**Pull request merged**